cluster/charts/rook-ceph/templates/clusterrolebinding.yaml

```diff
@@ -49,6 +49,19 @@ subjects:
 ---
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rbd-csi-nodeplugin
+subjects:
+  - kind: ServiceAccount
+    name: rook-csi-rbd-plugin-sa
+    namespace: {{ .Release.Namespace }} # namespace:operator
+roleRef:
+  kind: ClusterRole
+  name: rbd-csi-nodeplugin
+  apiGroup: rbac.authorization.k8s.io
```

Comment:
It seems that the Helm charts were missing the `ClusterRoleBinding` `rbd-csi-nodeplugin`, which I didn't notice as a discrepancy between the charts and `common.yaml` before.